src/Hocon.Tests.Performance/ConfigLookupSpec.cs

```diff
@@ -0,0 +1,140 @@
+using Hocon;
+using NBench;
+
+namespace Hocon.Tests.Performance
+{
+    public class ConfigLookupSpec
```

Comment:
Put this project into its own separate PR, 100% independently of any other changes so we can get a clean before / after snapshot.